test/src/sdk-harness/test_projects/logging/src/main.sw

```diff
@@ -17,7 +17,7 @@ fn main() {
     let k: b256 = 0xef86afa9696cf0dc6385e2c407a6e159a1103cefb7e2ae0636fb33d3cb2a9e4a;
     let a: str[4] = "Fuel";
     let b: [u8;
-    3] = [1, 2, 3];
+    3] = [1u8, 2u8, 3u8];
```

Comment:
Why do these 3 numbers need a type after them? The array is declared as `u8`. If anything I would expect possibly only declaring the first value as a `u8` similar to a vec in rust.

Reply:
We just get warnings at the moment. We should be able to inver the type directly through the array but I guess we currently don't. I will revert this file for now as it has nothing to do with this PR.